lib/loaders/loaders_with_authentication/gravity.js

```diff
@@ -9,6 +9,8 @@ export default (accessToken, userID, requestIDs) => {
   const gravityLoader = gravityLoaderWithAuthenticationFactory(gravityAccessTokenLoader)
 
   return {
+    collectionLoader: gravityLoader(id => `collection/${id}`, { user_id: userID }),
```

Comment:
Nice, I like being able to inject `user_id` params at the loader level, rather than needing to remember to do it each time you use it.

Reply:
Yeah totes, me too!